README.md

```diff
@@ -1,18 +1,29 @@
-=== AMP ===
-Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo, johnwatkins0, joshuawold, ryankienstra
-Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance, ux, seo, official
-Requires at least: 4.9
-Tested up to: 5.6
-Stable tag: 2.0.10
-License: GPLv2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Requires PHP: 5.6
+# AMP
 
-The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo,   johnwatkins0, joshuawold, ryankienstra
+Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance,   ux, seo, official  
+Requires at least: 4.9  
+Tested up to: 5.6  
+Stable tag: 2.0.10  
+License: GPLv2 or later  
+License URI: http://www.gnu.org/licenses/gpl-2.0.html  
+Requires PHP: 5.6  
 
-== Description ==
+![The Official AMP Plugin for WordPress](.wordpress-org/banner-1544x500.png)
 
-This official plugin from the AMP project enables AMP content publishing with WordPress in a way that is fully and seamlessly integrated with the standard mechanisms of the platform. The key features are the following:
+The official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+
+[![Build Status](https://github.com/ampproject/amp-wp/workflows/Build,%20test%20&%20measure/badge.svg)](https://github.com/ampproject/amp-wp/actions?query=branch%3Adevelop+workflow%3A%22Build%2C+test+%26+measure%22) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
+
+## Description
+
+This official plugin from the AMP project enables AMP content publishing with WordPress in a way that is fully and seamlessly integrated with the standard mechanisms of the platform.
+
+https://www.youtube.com/watch?v=s52JNMT59s8&list=PLXTOW_XMsIDRGRr5QDffrvND8Qh1RndFb
```

Comment:
Would this be embeddable on wp.org? GitHub doesn't render it appropriately:

![image](https://user-images.githubusercontent.com/16200219/105652084-6368ce00-5eb0-11eb-99c9-3e3251d69273.png)

Reply:
Yes, see https://make.wordpress.org/core/2010/02/20/plugins-can-now-include-videos-in-their-readme-txt-files/

It previews as expected on https://wpreadme.com/

![image](https://user-images.githubusercontent.com/134745/105653057-a787ae80-5e6f-11eb-8632-f9492678a785.png)

I've added videos like that before in readmes. Actually, `wp-dev-lib` supports that in transforming `readme.txt` to `readme.md` (at least in shortcode form):

https://github.com/xwp/wp-dev-lib/blob/a62ccca94f9995f294e12d500c45856adff81e34/scripts/class-wordpress-readme-parser.php#L97-L99

It's unfortunate that the bare URL in the `README.md` here would not be rendered on GitHub, but it seems like an acceptable tradeoff.

Reply:
We could could contribute up to the [`WordPressdotorg\Plugin_Directory\Markdown`](https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/plugins/plugin-directory/class-markdown.php) class the ability to transform something more markdown-friendly into the bare URL that would be the YouTube embed.

For example, let's say that the `README.md` had this line:

```md
[![Play video on YouTube](https://i1.ytimg.com/vi/s52JNMT59s8/hqdefault.jpg)](https://www.youtube.com/watch?v=s52JNMT59s8&list=PLXTOW_XMsIDRGRr5QDffrvND8Qh1RndFb)
```

This would be rendered on GitHub as:

[![Play video on YouTube](https://i1.ytimg.com/vi/s52JNMT59s8/hqdefault.jpg)](https://www.youtube.com/watch?v=s52JNMT59s8&list=PLXTOW_XMsIDRGRr5QDffrvND8Qh1RndFb)

But then the `Markdown` class could transform any such YouTube-linked images to unwrap top the bare URL:

```
https://www.youtube.com/watch?v=s52JNMT59s8&list=PLXTOW_XMsIDRGRr5QDffrvND8Qh1RndFb
```

Then on WordPress.org the oEmbed would render, whereas on GitHub there would be a poster image linked to the video on YouTube.

Reply:
That sounds like a great idea!

Reply:
Filed a ticket requesting this: https://meta.trac.wordpress.org/ticket/5582